examples/README.md

```diff
@@ -26,30 +26,30 @@ As an example, check out the TodoMVC example here: <https://examples.yew.rs/todo
-| [nested_list](nested_list)                          | Renders a styled list which tracks hover events                                                                                    |
-| [node_refs](node_refs)                              | Uses a [`NodeRef`](https://yew.rs/concepts/components/refs) to focus the input element under the cursor                       |
-| [pub_sub](pub_sub)                                  | Cross-component communication using [Agents](https://yew.rs/concepts/agents)                                                  |
-| [router](router)                                    | The best yew blog built with `yew-router`                                                                                          |
-| [store](store)                                      | Showcases the `yewtil::store` API                                                                                                  |
-| [timer](timer)                                      | Demonstrates the use of the interval and timeout services                                                                          |
-| [todomvc](todomvc)                                  | Implementation of [TodoMVC](http://todomvc.com/)                                                                                   |
-| [two_apps](two_apps)                                | Runs two separate Yew apps which can communicate with each other                                                                   |
-| [webgl](webgl)                                      | Controls a [WebGL canvas](https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Tutorial/Getting_started_with_WebGL) from Yew |
+| Example                                            | Description                                                                                                                        |
+| -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| [boids](boids)                                     | Yew port of [Boids](https://en.wikipedia.org/wiki/Boids)                                                                           |
+| [counter](counter)                                 | Simple counter which can be incremented and decremented                                                                            |
+| [dyn_create_destroy_apps](dyn_create_destroy_apps) | Uses the function `start_app_in_element` and the `AppHandle` struct to dynamically create and delete Yew apps                      |
+| [file_upload](file_upload)                         | Uses the `gloo::file` to read the content of user uploaded files                                                                   |
```

Comment:
Going to fix this the way it was

Reply:
Fixed, but i sorted this list in ascending order so that's why it looks so modified